chore: remove mentioning of not spec'ed Route Broadcasting Protocol (#597)
interledger/rfcs: 0031-dynamic-configuration-protocol/0031-dynamic-configuration-protocol.md

# Interledger Dynamic Configuration Protocol (ILDCP) v1

## Prerequisites

This specification assumes the reader is familiar with the following documents:

- [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md)
- [Interledger Protocol V4 (ILPv4)](../0027-interledger-protocol-4/0027-interledger-protocol-4.md)

## Terminology

- A **node** is a participant in an Interledger network. It may be a [connector](../0001-interledger-architecture/0001-interledger-architecture.md#connectors), a sender or a receiver.
- Nodes are connected to each other and the relationship between two nodes is either:
  - `parent` and `child` (one node is the `parent` of another node that is relatively its `child`), or,
  - `peer` and `peer` (two nodes are peered with one another).

## Overview

> Interledger is a protocol suite that consists of several protocols including the [Bilateral Transfer Protocol](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md) or [ILP over HTTP](../0035-ilp-over-http/0035-ilp-over-http.md) and others necessary for establishment, operation and maintenance of the network. Interledger Dynamic Configuration Protocol (ILDCP) is one of these protocols.

Author: Added ILP over HTTP and removed Route Broadcasting Protocol. I did wonder what "and others necessary" would include.

> Interledger is a protocol suite that consists of several protocols including the [Bilateral Transfer Protocol](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md) or [ILP over HTTP](../0035-ilp-over-http/0035-ilp-over-http.md) and others necessary for establishment, operation and maintenance of the network. Interledger Dynamic Configuration Protocol (ILDCP) is one of these protocols.

Reviewer: ```suggestion
Interledger is a protocol suite that consists of several protocols such as the [Bilateral Transfer Protocol](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md), [ILP over HTTP](../0035-ilp-over-http/0035-ilp-over-http.md) and others necessary for establishment, operation and maintenance of the network. Interledger Dynamic Configuration Protocol (ILDCP) is one of these protocols.
```
IMO "or" might be misleading here?

## Protocol Detail

### Procedure

An exchange of configuration information is done using the following procedure.

1. A child node requests configuration information from the parent node
2. The parent node responds with the configuration information to the child node
3. If the request cannot be processed, the parent node responds with an error

### Packet

> The request and the response above are transferred in [ILP packets](../0027-interledger-protocol-4/0027-interledger-protocol-4.md#specification).

Reviewer: ```suggestion
Requests and responses are transferred in [ILP packets](../0027-interledger-protocol-4/0027-interledger-protocol-4.md#specification).
```
